rotkehlchen/globaldb/cache.py

```diff
@@ -29,37 +35,47 @@ def globaldb_set_general_cache_values_at_ts(
     """Function to update cache in globaldb. Inserts all values paired with the cache key.
     If any entry exists, overwrites it."""
     cache_key = _compute_cache_key(key_parts)
+    unique_cache = any(cache_key.startswith(key_part) for key_part in UNIQUE_CACHE_KEYS)
+    if unique_cache:
+        cache = UNIQUE_CACHE
+    else:
+        cache = GENERAL_CACHE
     tuples = [(cache_key, value, timestamp) for value in values]
     write_cursor.executemany(
-        'INSERT OR REPLACE INTO general_cache '
+        f'INSERT OR REPLACE INTO {cache} '
```

Comment:
This can now raise a new error if there is a conflict with the primary key. Is something that needs to be handled in the 3 keys where it can happen. It should be `sqlcipher.IntegrityError`